src/wallet/wallet.cpp

```diff
@@ -2517,8 +2503,6 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
                     return false;
                 }
 
-                dPriority = wtxNew.tx->ComputePriority(dPriority, nBytes);
-
```

Comment:
@MarcoFalke I meant removing dead code in a commit titled as adding a warning is odd.